Assets/Shopify/UIToolkit/ShopControllers/BaseShopController.cs

```diff
@@ -42,6 +43,16 @@ public abstract class ShopControllerBase : MonoBehaviour, IShopCredentials {
         }
         [SerializeField] private string _accessToken;
 
+        /// <summary>
+        /// The Apple Merchant ID for processing Apple Pay payments.
+        /// </summary>
+        public string AppleMerchantID {
+            set { Cart.AppleMerchantID = value; } 
+            get { return Cart.AppleMerchantID; }
+        }
+        [Header("Payment Properties")]
+        [SerializeField] private string _appleMerchantID;
```

Comment:
the header attributes here serve no purpose because the custom editor overrides it